middlewares/userValidations.js

```diff
@@ -109,6 +109,46 @@ export default {
       return next(error);
     }
     next();
+  },
+  loginParamsValidator(req, res, next) {
+    const availableParams = (req.body.email)
```

Comment:
The params you should be checking for while logging in will be `usernameOrEmail` and `password` to rhyme with what is in the controller.

Reply:
okay... thanks